python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -109,26 +186,15 @@ def evaluate(metrics, y_true, y_pred, aggregate='mean'):
 
         res_list = []
         for metric in metrics:
-            if callable(metric):
-                metric_func = metric
-            else:
-                metric_func = TORCHMETRICS_REGRESSION_MAP[metric]
             if len(original_shape) in [2, 3] and aggregate is None:
-                res = torch.zeros(y_true.shape[-1])
+                res = np.zeros(y_true.shape[-1])
                 for i in range(y_true.shape[-1]):
-                    if callable(metric):
-                        res[i] = torch.from_numpy(metric_func(y_true[..., i], y_pred[..., i]))
-                    else:
-                        res[i] = metric_func(y_pred[..., i], y_true[..., i])
+                    res[i] = eval(metric)(y_true[..., i], y_pred[..., i])
```

Comment:
we should never use `eval` function since it might be dangerous and vulnerable. just make `REGRESSION_MAP` a dict e.g. `{'mae': mae, 'mse': mse, ...}`